react/src/pages/ErrorLogListPage.tsx

```diff
@@ -0,0 +1,203 @@
+import BAIModal from '../components/BAIModal';
+import Flex from '../components/Flex';
+import { RedoOutlined, DeleteOutlined } from '@ant-design/icons';
+import { Button, Space, Typography, Table, Alert } from 'antd';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+type logType = NonNullable<
+  {
+    isError: boolean;
+    message: string;
+    requestMethod: string;
+    requestParameters: string;
+    requestUrl: string;
+    statusCode: number;
+    statusText: string;
+    timestamp: string;
+    type: string;
+    title?: string;
+  }[]
+>;
+const ErrorLogListPage: React.FC = () => {
+  const { t } = useTranslation();
+  const [logs, setLogs] = useState<logType>(
+    JSON.parse(localStorage.getItem('backendaiwebui.logs') || '[]'),
```

Comment:
`localStorage.getItem` and JSON.parse run when every rendering. The easy wat to avoid this is using `useLocalStorageState`